content/articles/integrate-airtable-with-nextjs/index.md

```diff
@@ -1,10 +1,10 @@
-![Hero image](/hero-img.png)
+![Hero image](https://cloud-ed45a3xcq-hack-club-bot.vercel.app/1hero-img.png)
```

Comment:
Hey @giridhar7632. Section does not allow cdn images. We prefer all the images hosted locally. 
Please revert that change. Please refer to the [docs](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/UPLOAD_INSTRUCTIONS.md) and fix image links.

Finally, please run your article through a grammar checker like [Grammarly](https://www.grammarly.com/) to improve the quality of the article.